app/lib/common/enums/app_state.dart

```diff
@@ -0,0 +1 @@
+enum ViewState { Idle, Busy }
```

Comment:
My open PR may break this... my problem

Reply:
@holtzmak I tried  the code on that article and it worked perfectly. We don't even have use for ViewState anymore and can be removed.